docs/characters.md

```diff
@@ -164,10 +165,11 @@ This table holds vital static information for each character. It is used to crea
 [72]: #knownTitles
 [73]: #actionbars
 [74]: #grantablelevels
-[75]: #creation_date
-[76]: #deleteInfos_Account
-[77]: #deleteInfos_Name
-[78]: #deleteDate
+[75]: #order
+[76]: #creation_date
+[77]: #deleteInfos_Account
+[78]: #deleteInfos_Name
+[79]: #deleteDate
```

Comment:
rename them all small letters and no \_

Only need to do it here or the links will ve broken

Reply:
It is already in the list of updates of the character table, PR #584, thanks to @r-o-b-o-t-o  I took the description of the order column.